### PR TITLE
Feature/unit test

### PR DESCRIPTION
**Pull request by anoop1409**
1. Renamed `CustomUserCreatationForm` to `CustomUserCreationForm`
2. Added Pycharm config directory to `.gitignore`
3. Added unit test for the `CustomUserCreationForm`
4. Did some minor code cleanup

### CODE REVIEW

File: devresources/urls.py [hunk 1]
```diff
@@ -33,10 +33,10 @@
     path('', include('core.urls')),
 ]
 
-if settings.DEBUG==True:
+if settings.DEBUG:
```
Comment (mnamegaurav) [inline devresources/urls.py:36]: Do not need to add `DEBUG==True`, it works same as only writing `DEBUG` in `if` condition.

**Review by mnamegaurav — CHANGES_REQUESTED** [review]
Please keep all the tests in `tests.py` file as this project is not that large, there is no need to add another file, it can create confusion among beginner developers.

File: devresources/urls.py (hunk 1 above)
Comment (anoop1409) [reply devresources/urls.py:36]: @hamhaingaurav Isn't that what I did in my code?

**Comment by anoop1409** [comment]
This is updated

File: devresources/urls.py (hunk 1 above)
Comment (mnamegaurav) [reply devresources/urls.py:36]: Yup, sorry my bad, i wrote it in hurry.

**Comment by anoop1409** [comment]
Sorry. Closed this one by mistake.
Reopening the MR.

**Comment by mnamegaurav** [comment]
Amazing work, Thanks

**Pull request merged**